src/main/java/org/team1540/elmo/ejectors/BunnyEjectors.java

```diff
@@ -0,0 +1,54 @@
+package org.team1540.elmo.ejectors;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.team1540.elmo.Constants;
+
+public class BunnyEjectors extends SubsystemBase {
+
+    private final Solenoid suction;
+    private final Solenoid eject;
+
+    public BunnyEjectors(Solenoid suction,Solenoid eject) {
+        this.suction = suction;
+        this.eject = eject;
+        SmartDashboard.putNumber("LowerSuction/ejectSeconds",1);
+    }
+
+    public void setSuction(boolean sucking) {
+        suction.set(sucking);
+    }
+    public void setEject(boolean extended) {
+        eject.set(extended);
+    }
+    public void eject() {
+        eject.setPulseDuration(SmartDashboard.getNumber("Eject/ejectSeconds",1));
+        eject.startPulse();
+    }
```

Comment:
Calling it `eject` is misleading to me, doing something like `extend` would be more clear that it's the arm extension.

Reply:
done